iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/memtable/AbstractMemTable.java

```diff
@@ -609,6 +633,12 @@ public void deserialize(DataInputStream stream) throws IOException {
     totalPointsNumThreshold = stream.readLong();
     maxPlanIndex = stream.readLong();
     minPlanIndex = stream.readLong();
+    int databaseLength = stream.readInt();
```

Comment:
This seems likely to cause compatibility issues. Can we restore the database by some other means at recovery time without serializing it